scan-irb/report-en.tex

```diff
@@ -58,7 +58,23 @@
 \begin{description}[font=\normalfont,align=left,labelwidth=12em]
 \item [\textbf{\VAR{pat_name|e}}]
 \item [\textbf{Date of Birth}] \VAR{birth_date|e}
-\item [\textbf{Specimen}] Midnasal, self-collected
+\item [\textbf{Specimen}]
+  %- if swab_type == "mts"
+  Midnasal,
+  %- elif swab_type == "ans"
+  Anterior,
+  %- else
+  \errmessage{Invalid swab_type "\VAR{swab_type}"}
+  %- endif
+
+  %- if staff_collected == "1"
+  staff collected
+  %- elif staff_collected == "0"
+  self-collected
+  %- else
+  \errmessage{Invalid staff_collected "\VAR{staff_collected}"}
+  %- endif
```

Comment:
FYI, UWGreek samples are still self-collected (see Izzy's [comment](https://seattle-flu-study.slack.com/archives/C017M3GJ35X/p1594940377055900?thread_ts=1594940274.054800&cid=C017M3GJ35X)). Not sure if this is a change we want to add now or later.

Reply:
Yep. I believe the initial discussion with Izzy of changes needed was about self-vs-staff collection, but the final outcome was that the change is about staff-observed or not and how the specimen is transported back to the lab. Thus, I think we want a slightly different name than `staff_collected`.

Reply:
Great, let's discuss in [this comment thread](https://github.com/seattleflu/id3c-customizations/pull/119#discussion_r458441591).